docker/api.dockerfile

```diff
@@ -9,6 +9,10 @@ WORKDIR /usr/local/bin/src/literature
 
 ADD . .
 
-RUN pip install -r requirements.txt
+RUN pip3 install --upgrade pip
```

Comment:
You mentioned why you need to do this at standup but I didn't understand what the reason was

Reply:
Oh, I see the comments on scrum-blueteam. Everything makes sense